app/forms/introduce_yourself_form.rb

```diff
@@ -21,11 +21,18 @@ class IntroduceYourselfForm < Form
     message: "Make sure to answer this question",
   }
 
-  validates :birthday,
-    presence: { message: "Make sure to provide a birthday" }
+  validate :birthday_must_be_present_and_valid_date
 
-  # https://github.com/rails/rails/pull/8189#issuecomment-10329403
-  def class_for_attribute(attr)
-    return Date if attr == "birthday"
+  def birthday_must_be_present_and_valid_date
```

Comment:
It looks like some of this logic is duplicated at the model and controller level. Do you think we could extract this into a shared helper and potentially move the logic that's currently into the controller into a getter/setter for birthday on the model (/in the shared helper)?